css/style.css

```diff
@@ -1,3 +1,5 @@
 h1 {}
```

Comment:
This is doing nothing at the moment. Is that intentional?

Reply:
yes, sorry, old sample